src/amoc_controller.erl

```diff
@@ -292,3 +293,19 @@ get_test_status() ->
         [] -> finished;
         _Children -> running
     end.
+
+-spec does_scenario_exists(atom()) -> boolean().
+does_scenario_exists(Scenario) ->
```

Comment:
A typo here? I think the correct name should be `does_scenario_exist` without the ending `s`.